elasticsearch/examples/kubesphere/README.md

```diff
@@ -226,3 +159,84 @@ $ curl http://<your node ip>:30057/_cluster/health?pretty
   "active_shards_percent_as_number" : 100.0
 }
 ```
+
+### The reference configuration
+
+#### Master node
+
+```
+clusterName: "elasticsearch"
+nodeGroup: "master"
+
+roles:
+  master: "true"
+  ingest: "false"
+  data: "false"
+  ml: "false"
+  remote_cluster_client: "false"
+
+# tolerations for master node
```

Comment:
No need to add toleration for master node.

ES cluster should not be scheduled to master node in the production scenario.

You can comment out the toleration part for debug purpose